### ISSUE TITLE
Training process is sometimes frozen. 

### ISSUE THREAD

**Issue by MCDM2018** [junyanz/pytorch-CycleGAN-and-pix2pix#659]
Hello guys
Above all, thanks to authors.
now, i m using cyclegan for my project. but Sometimes the training process is frozen in epoch 72. Anyone experiences this situation?

**Comment by BoyaWang** [comment]
Hi, I have the same problem. According to former answers, I use the command below. 
python train.py --dataroot ./datasets/cityscapes/ --name cityscapes_pix2pix --model pix2pix --direction BtoA --display_id -1 --continue_train
I think the thing is to disable visdom (does not work on my devices), and use the label "--continue_train", sometimes it will get stuck again, but I try to replay the command, and check saved files in /checkpoints, it seems that all the epochs are recorded. Hope this can help.

**Comment by junyanz** [comment]
Starting the visdom by yourself before you run the program might help.

**Comment by BoyaWang** [comment]
Thanks for remind, yes, actually I tried, but still failed. It might be because of some kind of firewall that can block the visdom. I don't know exactly why. But once turn off visdom it should work.

**Comment by mrgloom** [comment]
But why it freeze at epoch 72?
